packages/adminrouter/extra/src/includes/main/common.conf

```diff
@@ -3,7 +3,11 @@
 # * some of the log messages that are required by test
 #   harness are not important enough to justify anything
 #   higher than info level.
-error_log stderr info;
+# Use a non-blocking Unix domain socket. If journald fails to read
+# the socket quickly enough, log messages may be lost, but this is
+# better than logging to a blocking connection (e.g. stderr) which
+# may cause Admin Router to hang.
+error_log syslog:server=unix:/dev/log info;
```

Comment:
Did you talk to @timaa2k about the syslog config, like about setting a specific facility?

Reply:
We are setting `access_log` in `extra/src/includes/http/common.conf` this way:
`access_log syslog:server=unix:/dev/log,facility=daemon customformat;`

Any reason not to set the `error_log` similarly? 
CC: @timaa2k

Reply:
D'oh! I even noted this in the JIRA issue, but then forgot to add it to the code.